docs-ui/components/doDont.tsx

```diff
@@ -0,0 +1,130 @@
+import {Component} from 'react';
+import {styled} from '@storybook/theming';
+
+import {IconCheckmark, IconClose} from 'app/icons';
+import theme from 'app/utils/theme';
+
+type BoxContent = {
+  text: string;
+  img: {
+    src: string;
+    alt: string;
+  };
+};
+type Props = {
+  boxes: {
```

Comment:
is `boxes` helpful here? I think having `do`/`dont` as top level props is sufficient.